.travis.yml

```diff
@@ -21,6 +25,14 @@ env:
   global:
     - CFLAGS="-Werror -Wall -Wextra -Wno-error=sign-compare -Wno-error=unused-parameter -Wno-error=missing-field-initializers"
 
+before_install:
+  - |
+    if [ "$TRAVIS_OS_NAME" == freebsd ]; then
+      sudo pkg install -y evdev-proto libgudev libxml++
+      sudo pkg install -y automake libtool pkgconf gmake
+      sudo ln -sf $(which gmake) $(which make)
```

Comment:
I forgot: aren't we root anyway? do we even need sudo here? it's used elsewhere already but might be unnecessary there too.

Reply:
I've tried: `pkg install` failed without `sudo`. [Official examples](https://docs.travis-ci.com/user/installing-dependencies/) also use `sudo`.

Reply:
oki, thanks for testing